Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -128,38 +133,141 @@ static async Task<IEvent> PlatformGetEventByIdAsync(string eventId)
+                    Attendees = GetAttendeesForEvent(eventId),
+                    RecurrancePattern = !string.IsNullOrEmpty(rRule) ? GetRecurranceRuleForEvent(rRule) : null
+                };
             }
             catch (NullReferenceException)
             {
                 throw new NullReferenceException($"[Android]: No Event found for event Id {eventId}");
             }
         }
+
+        static IReadOnlyList<IAttendee> GetAttendeesForEvent(string eventId)
+        {
+            var attendeesUri = CalendarContract.Attendees.ContentUri;
+            var attendeesProjection = new List<string>
+            {
+                CalendarContract.Attendees.InterfaceConsts.EventId,
+                CalendarContract.Attendees.InterfaceConsts.AttendeeEmail,
+                CalendarContract.Attendees.InterfaceConsts.AttendeeName
+            };
+            var attendeeSpecificAttendees = $"{CalendarContract.Attendees.InterfaceConsts.EventId}={eventId}";
+            var cur = Platform.AppContext.ApplicationContext.ContentResolver.Query(attendeesUri, attendeesProjection.ToArray(), attendeeSpecificAttendees, null, null);
+            var attendees = new List<IAttendee>();
+            while (cur.MoveToNext())
+            {
+                attendees.Add(new Attendee()
+                {
+                    Name = cur.GetString(attendeesProjection.IndexOf(CalendarContract.Attendees.InterfaceConsts.AttendeeName)),
+                    Email = cur.GetString(attendeesProjection.IndexOf(CalendarContract.Attendees.InterfaceConsts.AttendeeEmail)),
+                });
+            }
+            return attendees.AsReadOnly();
+        }
+
+        static RecurrenceRule GetRecurranceRuleForEvent(string rule)
+        {
+            var recurranceRule = new RecurrenceRule();
+            if (rule.Contains("FREQ="))
+            {
+                var ruleFrequency = rule.Substring(rule.IndexOf("FREQ=", StringComparison.Ordinal) + 5);
+                ruleFrequency = ruleFrequency.Contains(";") ? ruleFrequency.Substring(0, ruleFrequency.IndexOf(";")) : ruleFrequency;
+                switch (ruleFrequency)
+                {
+                    case dailyFrequency:
+                        recurranceRule.Frequency = RecurrenceFrequency.Daily;
+                        break;
+                    case weeklyFrequency:
+                        recurranceRule.Frequency = RecurrenceFrequency.Weekly;
+                        break;
+                    case monthlyFrequency:
+                        recurranceRule.Frequency = RecurrenceFrequency.Monthly;
+                        break;
+                    case yearlyFrequency:
+                        recurranceRule.Frequency = RecurrenceFrequency.Yearly;
+                        break;
+                }
+            }
+
+            if (rule.Contains("INTERVAL="))
+            {
+                var ruleInterval = rule.Substring(rule.IndexOf("INTERVAL=", StringComparison.Ordinal) + 9);
```

Comment:
We should move all of the string literals into consts. 

it would be good to move the numbers like `9` and `6`  as well so they have significance - right now they just random numbers being added to the index (not sure why)

Reply:
Dealt with all statics as mentioned above, the ints are simply string lengths, so now they will be referenced correctly as they are actual strings.